var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -180,6 +180,13 @@ def configure_args(self):
                                                 spec else 'yes')
         ]
 
+        if '+slurm' in spec:
+            config_args.append('--with-slurm=yes')
+            config_args.append('--with-slurm-include={0}'.format(spec['slurm'].prefix.include))
```

Comment:
Will this actually account for `include/slurm` in this form?

Reply:
No, just `include/`. I do not think that `include/slurm` would be correct. All clean C/C++ projects prefix their includes with their name to avoid naming collisions (e.g. for system-wide installs).

The include dir for slurm is still include/ and downstream projects use it as <slurm/something.hpp>.